test/Tests/src/Data/Text/Utils_Spec.enso

```diff
@@ -89,4 +89,15 @@ spec =
             Text_Utils.take_suffix (kshi+kshi+'a'+kshi) 2 . should_equal 'a'+kshi
             Text_Utils.take_suffix (kshi+kshi+'a'+kshi) 1 . should_equal kshi
 
+    Test.group "to_display_text" <|
+        Test.specify "simple conversion" <|
+            "Hello".to_display_text . should_equal "Hello"
+
+        Test.specify "long text conversion" <|
+            long = "Hello World! ".repeat 1024
+            disp = long.to_display_text
+            disp.length . should_equal 77
+            disp.characters.take (First 5) . should_equal [ 'H', 'e', 'l', 'l', 'o' ]
+            disp.characters.take (Last 6) . should_equal [ 'W', 'o', 'r', 'l', 'd', '!' ]
```

Comment:
I think we should add a test where we use multi-codepoint grapheme clusters, ideally such that the default split would end up in the middle of it, to ensure that the clusters are not broken.

Reply:
Can you give me an example of a string you'd like me to test?

Reply:
Sure, let's do three (but I care most about the first one):
```
'a\u0321\u0302'*100
'\u0915\u094D\u0937\u093F'*100
'\u{1F926}\u{1F3FC}\u200D\u2642\uFE0F'*100
```

Reply:
They are all 100 graphemes in length (the thing I put in `''` is always a single grapheme cluster here)

Reply:
Added the tests in 4bc2353